src/website/app/demos/Dialog/examples/Dialog/stacking.js

```diff
@@ -0,0 +1,99 @@
+/* @flow */
+import { Component } from 'react';
+import Button from '../../../../../../library/Button';
+import Dialog from '../../../../../../library/Dialog';
+
+export default {
+  id: 'stacking',
+  title: 'Stacking',
+  description: `This is broken in at least two ways:
+
+1. Only the top-most overlay should display
+2. Closing a stacked Dialog turns off focus trapping rather than adjusting its
+   target`,
```

Comment:
Good idea to add these notes directly to the example @kylegach :) I also had a note that opening the second dialog jumps the hidden/overlaid app to the top of the page, do you know if that's still the case?

Reply:
Yes, it does. I had forgotten that one; thanks.